launch

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import json
+import atexit
+import signal
+import argparse
+import subprocess
+
+from distutils.version import StrictVersion
+from subprocess import run, Popen
+
+
+DOCKER_IMAGE = 'claraparabricks/single-cell-examples_rapids_cuda10.2:v0.0.2'
+
+# Key: String: Example name
+# Value: String: Comma seperated HTTP URL to dataset files.
+DATASETS = {
+    'hlca_lung': 'https://rapids-single-cell-examples.s3.us-east-2.amazonaws.com/krasnow_hlca_10x.sparse.h5ad',
+    'dsci_bmmc_60k': """
+            https://rapids-single-cell-examples.s3.us-east-2.amazonaws.com/dsci_resting_nonzeropeaks.h5ad,
+            https://rapids-single-cell-examples.s3.us-east-2.amazonaws.com/dsci_resting_peaknames_nonzero.npy,
+            https://rapids-single-cell-examples.s3.us-east-2.amazonaws.com/dsci_resting_cell_metadata.csv
+            """,
+    'dsci_bmmc_60k_viz': """
```

Comment:
Why is the same dataset here under two names (dsci_bmmc_60k and dsci_bmmc_60k_viz)?

Reply:
Currently the script config data model is 
            Example(Notebook)  
                      |->  Conda Env(Many to One)
                      |->  DataSet (one to one)

Since, the relation between Examples and Datasets is one to one(in the current version), we have duplicate dataset.

In a future version we can name a dataset and change the relationship between Example to Dataset  Many to Many. At the moment I felt it will be an overkill for just one example.

Reply:
I see, thanks! But the dataset for the viz notebook is not this one. It's the hlca_lung dataset.

Reply:
This was a blunder. Fixed commited.